compose/config/compose_spec.json

```diff
@@ -92,7 +92,8 @@
                 "target": {"type": "string"},
                 "shm_size": {"type": ["integer", "string"]},
                 "extra_hosts": {"$ref": "#/definitions/list_or_dict"},
-                "isolation": {"type": "string"}
+                "isolation": {"type": "string"},
+                "extra_cli_build_args": {"$ref": "list_of_strings"}
```

Comment:
Hi! Thank you for the contribution. Please propose a PR to add this new property in the Compose specification schema at https://github.com/compose-spec/compose-spec/blob/master/schema/compose-spec.json. 

This file is downloaded from there and we try to avoid having changes that are not coming from compose-spec.

Reply:
https://github.com/compose-spec/compose-spec/pull/145

Reply:
Is there a way to run the tests in CI against this branch of compose-spec?